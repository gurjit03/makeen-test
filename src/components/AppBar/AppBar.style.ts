import styled from 'styled-components'
import { colors, verticalSpacing } from '../../config/jss'

export const Wrapper = styled.header`
	display: flex;
	align-items: center;
	height: 56px;
	background-color: ${colors.primary};
	padding-left: ${verticalSpacing}px;
`
