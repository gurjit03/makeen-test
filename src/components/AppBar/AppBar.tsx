import React from 'react'
import CompanyLogo from '../CompanyLogo/CompanyLogo'
import { Wrapper } from './AppBar.style'

const AppBar: React.FC = () => (
	<Wrapper>
		<CompanyLogo />
	</Wrapper>
)

export default AppBar
