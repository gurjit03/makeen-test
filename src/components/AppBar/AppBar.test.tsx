import React from 'react';
import { render } from '@testing-library/react'
import AppBar from './AppBar';

describe('<AppBar />', () => {
    it('should render properly', () => {
        const { asFragment } = render(<AppBar />)
        expect(asFragment()).toMatchSnapshot();
    });
})