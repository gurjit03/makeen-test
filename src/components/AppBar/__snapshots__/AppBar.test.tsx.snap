// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppBar /> should render properly 1`] = `
<DocumentFragment>
  <header
    class="sc-gsTEea sIHru"
  >
    <h3
      class="sc-bdfBQB epGijZ"
    >
      COMATCH
    </h3>
  </header>
</DocumentFragment>
`;
